### ISSUE TITLE
Gpt-J

### ISSUE THREAD

**Issue by robitec97** [minimaxir/aitextgen#223]
Can i train gpt-j from scratch?

**Comment by Vectorrent** [comment]
Yes. It's not going to work out-of-the-box with AITextGen, but you should be able to make it work, with a little bit of tweaking. GPT-J is a part of the Huggingface Transformers module, so training should be nearly identical to how you'd train GPT-2/GPT-Neo with AITextGen.

**Comment by breadbrowser** [comment]
> Can i train gpt-j from scratch?

yes
